content/reference-amendments.md

```diff
@@ -23,22 +23,21 @@ The following is a comprehensive list of all known amendments and their status o
 | [PayChan][]               | v0.33.0    | [Enabled: 2017-03-31](https://xrpcharts.ripple.com/#/transactions/16135C0B4AB2419B89D4FB4569B8C37FF76B9EF9CE0DD99CCACB5734445AFD7E "BADGE_GREEN") |
 | [Escrow][]                | v0.60.0    | [Enabled: 2017-03-31](https://xrpcharts.ripple.com/#/transactions/C581E0A3F3832FFFEEB13C497658F475566BD7695B0BBA531A774E6739801515 "BADGE_GREEN") |
```

Comment:
You know that I know that you can sort things, probably with your eyes closed LOL, so my questions are just about understanding the sort order being used. If the Status is the first sort, is the Introduced version the second sort, with the older version coming first? Or is the Name the second sort? For PayChan and Escrow above, the former is true. For FlowCross and SHAMapV2 at the top, the latter is true.